src/gatling/simulations/BasicSimulation.scala

```diff
@@ -0,0 +1,87 @@
+import com.google.auth.oauth2.GoogleCredentials
+import java.io.ByteArrayInputStream
+import java.util.UUID
+
+import com.typesafe.config.{Config, ConfigFactory}
+import io.gatling.http.Predef._
+import io.gatling.core.Predef._
+import scala.collection.JavaConverters._
+import sys.process._
+
+class BasicSimulation extends Simulation {
```

Comment:
We talked about keeping as much of the code in java as possible and importing it into the Simulation file, making the Simulation file as light and bare bones as possible. Can we attempt to do that here?

Reply:
Yes, let's use the current POC to discuss what needs to be refactored into Java.